include/host1x.h

```diff
@@ -161,12 +161,12 @@ int host1x_overlay_set(struct host1x_overlay *overlay,
 
 #define HOST1X_BO_CREATE(host1x, size, flags)				\
 ({									\
-	struct host1x_bo *bo = host1x_bo_create(host1x, size, flags);	\
-	if (!bo)							\
+	struct host1x_bo *_bo_ = host1x_bo_create(host1x, size, flags);	\
+	if (!_bo_)							\
 		fprintf(stderr,						\
 			"ERROR: %s:%d: host1x_bo_create() failed\n",	\
 			__FILE__, __LINE__);				\
-	bo;								\
+	_bo_;								\
 })
```

Comment:
In case of opentegra, it fallbacks to non-acceleratable allocation. I don't mind grate tests crashing, it's likely to be something much more severe than OOM.